Added error string for process operations

rajasec opened the pull request:
Added error strings when there is no process operations.

Signed-off-by: rajasec <rajasec79@gmail.com>

hqhq commented on libcontainer/error.go:
```diff
@@ -46,6 +46,8 @@ func (c ErrorCode) String() string {
 	case ConsoleExists:
 		return "Console exists for process"
+	case ProcessNotExecuted:
+		return "No process operations"
```
The name and message seems not well matched, can we change `ProcessNotExecuted` to something like `NoProcessOps`? The name `ProcessNotExecuted` seems has more meaning than on ops.

rajasec replied:
@hqhq 
I've fixed and committed. Please review

crosbymichael commented:
@rajasec why did you not keep `ProcessNotExecuted` and just added the string for it?

rajasec commented:
@crosbymichael 
Whenever ProcessNotExecuted error occurs, string printed will be "uknown error" so I have added the error string to show there is no process operations.
From the review comments I've changed the error name to NoProcessOps instead of ProcessNotExecuted
Let me know if you are OK.

hqhq commented:
Consider what we have for errors:
```
   9 const (
 10         // Factory errors
 11         IdInUse ErrorCode = iota
 12         InvalidIdFormat
 13
 14         // Container errors
 15         ContainerNotExists
 16         ContainerPaused
 17         ContainerNotStopped
 18         ContainerNotRunning
 19
 20         // Process errors
 21         ProcessNotExecuted
 22
 23         // Common errors
 24         ConfigInvalid
 25         ConsoleExists
 26         SystemError
 27 )
```
For process errors, I think all kinds of process errors could cause process not executed, so we may need several specific errors to show more detailed information like other errors, so `NoProcessOps` could be a start. @crosbymichael WDYT?

rajasec commented:
@crosbymichael 
Please let me know your comments ?

crosbymichael commented:
Ok, sounds good.  @rajasec can you rebase?

rajasec commented:
@crosbymichael 
I'm done with rebase.

hqhq commented:
LGTM

crosbymichael commented:
LGTM
